site/docs/contribute/environment.md

```diff
@@ -46,3 +46,53 @@ a docker registry as a proxy for all public images. This registry runs in a dock
 export DOCKERHUB_USERNAME='your docker hub username'
 export DOCKERHUB_PASSWORD='your password'
 ```
+
+## Optional Code Improvement and Verification Tools
+
+Fybrik repositories use different linters to validate and improve code.  
+*golangci-lint* works as a Go linters aggregator, which includes a lot of linter such as `staticcheck`, `revive`, `goimports` and more.  
+You can check the configuration to each repository in `.golangci.yml`.
+
+
+## Integrating golangci-lint with VS Code
+Change the default lint to golanci-lint in VS Code:
+
+1. Install golangci-lint: [https://golangci-lint.run/usage/install/](https://golangci-lint.run/usage/install/)
+2. Open VS Code `setting.json`:
+    1. Open the Command Palette: `Ctrl+Shift+P` 
+    2. Search for "Open Settings (JSON)"
```

Comment:
where to search? in the repository? in the VC installation  ?

Reply:
When you press Ctrl+Shift+P a search box for commands is opened